task10.py

```diff
@@ -0,0 +1,15 @@
+#Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+
+#Пример:
+# [2, 3, 4, 5, 6] => [12, 15, 16];
+# [2, 3, 5, 6] => [12, 15]
+
+def mult_lst(lst):
+    l = len(lst)//2 + 1 if len(lst) % 2 != 0 else len(lst)//2
```

Comment:
```suggestion
    l = (len(lst) + 1) // 2
```
Так будет проще, не правда ли?